app/uk/gov/hmrc/cgtpropertydisposals/repositories/enrolments/VerifiersRepository.scala

```diff
@@ -39,13 +42,24 @@ trait VerifiersRepository {
 }
 
 @Singleton
-class DefaultVerifiersRepository @Inject() (mongo: MongoComponent)(implicit
+class DefaultVerifiersRepository @Inject() (mongo: MongoComponent, config: Configuration)(implicit
   ec: ExecutionContext
 ) extends PlayMongoRepository[UpdateVerifiersRequest](
       mongoComponent = mongo,
       collectionName = "update-verifiers-requests",
       domainFormat = UpdateVerifiersRequest.format,
-      indexes = Seq(IndexModel(ascending("ggCredId"), IndexOptions().name("ggCredIdIndex")))
+      indexes = Seq(
+        IndexModel(
+          ascending("ggCredId"),
+          IndexOptions()
+            .name("ggCredIdIndex")
+            .expireAfter(
+              config.get[FiniteDuration]("mongodb.verifiers-cache-ttl.expiry-time").toSeconds,
```

Comment:
We seem to be converting to seconds but have time unit configured as hours, which is incorrect. 
Also, `CacheRepository` internally handles the relevant time unit conversions. So, we shouldn't set anything here on our end. As a result, I would just change it to following
`expireAfter(config.get[FiniteDuration]("mongodb.verifiers-cache-ttl.expiry-time"))`